test/e2e/tests/test_endpoint.py

```diff
@@ -218,6 +218,7 @@ def faulty_config(name_suffix, single_container_model):
 
 
 @service_marker
+@pytest.mark.shallow_canary
```

Comment:
This marker means if we run pytest with this marker then run this test. In your shell-script you are still running these tests with the regions in this `^(eu-south-2|ap-southeast-3|me-central-1|eu-central-2)` Correct if I'm wrong but isn't the goal of this pr to not run these tests in those regions since image is missing?

Reply:
We want to run shallow canary, which includes training and hosting, in eu-south-2, ap-southeast-3, me-central-1, eu-central-2. Training and hosting does not require missing images such as xgboost, debugger and clarify.